sections/06_discussion.md

```diff
@@ -198,6 +198,124 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
 machine that requires manual validation? DeepChem and DragoNN are worth
 discussing here.*
 
-### Transfer learning/transferability of features
+### Multimodal and transfer learning
 
 * https://github.com/greenelab/deep-review/issues/139#issuecomment-268901804
```

Comment:
I think it's currently discussed here:
https://github.com/greenelab/deep-review/blame/master/sections/03_categorize.md#L125

Probably the best thing to do is to mention in that paragraph that we talk about transfer learning more fully in the discussion. @alxndrkalinin - do you want to make that edit to categorize?